### PR TITLE
Assign all tasks on date, on the task quick assign modal does not work

### PR DESCRIPTION
**Pull request by anant00729**
Assign all tasks on a date, on the task quick assign modal does not work

To test:
1. Go to http://localhost:3000/tasks
2. In the list of tasks, add two or more unassigned tasks for the same date.
3. click on unassigned, change it to some assignee and then click on the checkbox "Assign all unassigned tasks on this date to this person". then click on update.
4. You'll be able to see that the unassigned task will get updated with the assignee name for the same date.

For more details please check. 
https://lite-farm.atlassian.net/browse/LF-2314

### CODE REVIEW

**Review by jagDhillon077 — APPROVED** [review]
Functionality is working

File: packages/webapp/src/components/Modals/QuickAssignModal/index.jsx [hunk 1]
```diff
@@ -48,6 +65,11 @@ export default function TaskQuickAssignModal({
     dismissModal();
   };
 
+  const onCheckedAll = () => {
+    checkUnassignedTaskForSameDate();
```
Comment (smattingly) [inline packages/webapp/src/components/Modals/QuickAssignModal/index.jsx:69]: Is there a reason for this statement? The return value is not used, and I don't see a side effect from the call.

**Review by smattingly — CHANGES_REQUESTED** [review]
This looks good. I had one suggestion/question (see comment).

File: packages/webapp/src/components/Modals/QuickAssignModal/index.jsx (hunk 1 above)
Comment (anant00729) [reply packages/webapp/src/components/Modals/QuickAssignModal/index.jsx]: I have removed that, I was calling that function on the checkbox click, to test out the function's results.

**Review by smattingly — APPROVED** [review]

**Pull request merged**